INSTALL.md

```diff
@@ -222,3 +222,38 @@ For more information on installing/updating/uninstalling the RPM, check out our
 ## Debian Package
 
 Additional information on how to build a Debian package can be found in [dist/debian/DEBIAN_PACKAGE.md](dist/debian/DEBIAN_PACKAGE.md).
+
+## Run E2E tests
+
+The test suite is heavily relying on Docker Hub registry, since the introduction
+of the rate pull limit, developpers can quickly hit the quota limit leading to
+the e2e tests randomly failed.
+
+There is two possible approaches to minimize/avoid that:
+
+1. if you have an account on Docker Hub you can specify and export your
+credentials via environment variables `E2E_DOCKER_USERNAME` and
+`E2E_DOCKER_PASSWORD` before running the test suite, however if you have
+a free account the quota limit is simply doubled and may not work for you
+2. or you can run a local pull through cache registry and use
+`E2E_DOCKER_MIRROR`/`E2E_DOCKER_MIRROR_INSECURE` environment variables
+
+### Run a local pull through cache registry
+
+The most straightforward way to run it is to run in a terminal:
+
+```sh
+mkdir -p $HOME/.cache/registry
+apptainer run --env REGISTRY_HTTP_ADDR=127.0.0.1:5001 \
```

Comment:
```suggestion
singularity run --env REGISTRY_HTTP_ADDR=127.0.0.1:5001 \
```

Reply:
The new name isn't available yet so I think it's better to leave that for a later global change.